.swiftlint.yml

```diff
@@ -2,3 +2,4 @@ identifier_name:
   excluded:
     - id
     - dt
+    - q
```

Comment:
just a remark. clarity over brevity.

https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingBasics.html

Reply:
Follow you. Thanks.

Reply:
and Fixed